src/main/java/refinedstorage/item/ItemUpgrade.java

```diff
@@ -58,6 +63,8 @@ public static ItemStack getRequirement(int type) {
                 return new ItemStack(Blocks.CRAFTING_TABLE);
             case ItemUpgrade.TYPE_INTERDIMENSIONAL:
                 return new ItemStack(Items.NETHER_STAR);
+            case ItemUpgrade.TYPE_SILK_TOUCH:
+                return Items.ENCHANTED_BOOK.getEnchantedItemStack(new EnchantmentData(Enchantment.getEnchantmentByID(33), 1));
```

Comment:
You should use `Enchantment.getEnchantmentByLocation("silk_touch")` here as the id might change for what ever reason.